Spec/Auth.swift

```diff
@@ -234,15 +234,15 @@ class Auth : QuickSpec {
                     let channel = realtime.channels.get("test")
 
                     waitUntil(timeout: testTimeout) { done in
-                        realtime.connect()
-                        channel.publish("message", data: nil) { error in
-                            guard let error = error else {
+                        realtime.connection.on(.failed) {stateChange in
```

Comment:
This test doesn't pass like it was before? Just to be sure that we aren't changing tests that were validated (working the right way) and are important to give us confidence.

Reply:
done 
https://github.com/ably/ably-ios/pull/757/commits/b64024f8a23ed5524fc171cb4f63101f8fcb9f3c